les/odr_requests.go

```diff
@@ -307,6 +308,11 @@ type HelperTrieResps struct { // describes all responses, not just a single one
 	AuxData [][]byte
 }
 
+type GatewayFeeInformation struct {
```

Comment:
Why is this `struct` defined here? Because it is not used in this file, it seems like there is probably a better place for it.